mir-module/src/main/resources/xsl/metadata/mir-abstract.xsl

```diff
@@ -88,6 +88,17 @@
             <xsl:with-param name="query" select="concat('&amp;fq=link:*',$accessCondition, '&amp;owner=createdby:', $owner)" />
           </xsl:call-template>
         </xsl:if>
+          <xsl:if test="not(mcrxsl:isCurrentUserGuestUser())">
```

Comment:
because the badge on result list is also only shown for logged in users. Guests should only see published documents.